server/lib/queries.js

```diff
@@ -328,6 +329,11 @@ export const usersToNotifyForUpdateSQLQuery = `
         collective."ParentCollectiveId" IS NOT NULL
         AND m."CollectiveId" = collective."ParentCollectiveId"
         AND m."role" IN ('ADMIN', 'MEMBER')
+      ) OR (
+        --- Include child collective's contributors
+        c."ParentCollectiveId" IS NOT NULL
+        AND c.id = m."CollectiveId"
+        AND m."role" IN ('BACKER', 'ATTENDEE')
```

Comment:
Why invent something new instead of using `targetRoles`?

Reply:
Makes sense. I've changed that. 👍🏽